doc/book/examples.md

```diff
@@ -171,6 +171,7 @@ class CacheListener implements ListenerAggregateInterface
     use ListenerAggregateTrait;
 
     private $cache;
+    private $listeners = [];
```

Comment:
It shouldn't be there. The property is defined in trait `ListenerAggregateTrait`. Please remove.